config/crd/bases/pipelines.openshift.io_gitopsservices.yaml

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
(this comment is not about this file)
We may want to rename the following crd files to remove the `_crd` suffix.
* argoprojio_applicaitons_crd.yaml
* argoprojio_appliationsets_crd.yaml
* argoprojio_appprojects_crd.yaml
* argoprojio_argocds_crd.yaml